maps/away/ships/coc/coc_scarab/coc_scarab.dm

```diff
@@ -0,0 +1,178 @@
+/datum/map_template/ruin/away_site/coc_scarab
+	name = "Scarab Salvage Ship"
+	description = "Scarab salvage ship."
+	suffixes = list("ships/coc/coc_scarab/coc_scarab.dmm")
+	sectors = list(SECTOR_COALITION, SECTOR_WEEPING_STARS, SECTOR_ARUSHA, SECTOR_LIBERTYS_CRADLE)
+	spawn_weight = 1
+	ship_cost = 1
+	id = "coc_scarab"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/scarab_gas_harvester, /datum/shuttle/autodock/overmap/scarab_shuttle)
+
+	unit_test_groups = list(1)
+
+/singleton/submap_archetype/coc_scarab
+	map = "Scarab Salvage Ship"
+	descriptor = "Scarab salvage ship."
+
+/obj/effect/overmap/visitable/ship/coc_scarab
+	name = "Scarab Salvage Ship"
+	class = "SFV" //Scarab Fleet Vessel
+	desc = "An ancient Burrow-class transport vessel, bearing signs of extensive modification. These vessels are rarely seen in service outside of the Scarab Fleet in the modern day."
+	icon_state = "freighter"
+	moving_state = "freighter_moving"
+	colors = list("#a400c1", "#4d61fc")
+	designer = "Coalition of Colonies, Scarab Fleet"
+	volume = "57 meters length, 48 meters beam/width, 12 meters vertical height"
+	drive = "Low-Speed Warp Acceleration FTL Drive"
+	weapons = "Starboard fore ballistic armament, dual port flight craft."
+	sizeclass = "Modified Burrow-class transport."
+	shiptype = "Salvage, fuel extraction and mineral exploration."
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	invisible_until_ghostrole_spawn = TRUE
+	initial_generic_waypoints = list(
+		"scarab_nav1",
+		"scarab_nav2",
+		"scarab_nav3",
+		"scarab_nav4",
+		"scarab_dock1",
+		"scarab_dock2",
+		"scarab_dock3",
+		"scarab_dock4",
+		"scarab_dock5"
+	)
+	initial_restricted_waypoints = list(
+		"Intrepid" = list("scarab_intrepid"),
```

Comment:
no docks restricted to shuttles - this should just be a normal dock that any shuttle can use (exceptions of course being their hangars or starting docks, or events, or etc)